pkg/logging/logging_test.go

```diff
@@ -157,3 +158,18 @@ func TestAddUnrecognizedHook(t *testing.T) {
 	err := logger.AddHook("unrecognized_type", "some_destination")
 	Assert(t).IsNotNil(err, "Expected an error for adding an unrecognized hook output type")
 }
+
+func TestConcurrent(t *testing.T) {
```

Comment:
since this test has no assertions, I assume the point is simply that without the attached code change, it will crash due to a concurrent write?

I suggest this be noted somewhere so that others don't get confused about "hmm, no assertions, what is this testing?"